### PR TITLE
Add n8n to catalog

### PR DESCRIPTION
**Pull request by ericgaspar**
* Add n8n to catalog

### CODE REVIEW

**Review by tituspijean — APPROVED** [review]

**Comment by fflorent** [comment]
Can you tell me what do you think of this issue?

https://github.com/YunoHost-Apps/n8n_ynh/issues/6 (*Not totally free license*)

If I quote [your documentation](https://yunohost.org/en/contribute/packaging_apps/quality_tests/yep):
> The application list apps.json only accept packages with a free license, as well as the license for the contained application. Some free applications require non-free dependencies (example: MP3, drivers, etc.). In this case, you should add &dep-non-free to the acronym and if possible give details in the README.md of the package, in this case the integration will be accepted on a case-by-case basis.

I don't know whether it would be compatible with the apps.json file here.

**Comment by tituspijean** [comment]
That's a valid remark! I think we cannot allow such license in the catalog. 😞

**Comment by Limezy** [comment]
>  😞 Maybe it is time to revive the non-free apps list.

@tituspijean I think I found the exact same problem for my Outline app... 
Its licence is "Business Source License 1.1" (https://github.com/outline/outline/blob/main/LICENSE)

Do you confirm that I can't add it to the app catalog ? 😢 😭

**Comment by ericgaspar** [comment]
@tituspijean I propose to merge this with the addition of the licence antifeature

**Comment by fflorent** [comment]
LGTM

**Comment by nsakibp** [comment]
Please add

File: apps.json [hunk 1]
```diff
@@ -2007,6 +2007,14 @@
         ],
         "url": "https://github.com/YunoHost-Apps/mytinytodo_ynh"
     },
+    "n8n": {
+        "antifeatures": [
+            "Not totally free upstream"
```
Comment (tituspijean) [inline apps.json:2012]:
```suggestion
            "not-totally-free"
```

**Review by tituspijean — CHANGES_REQUESTED** [review]

**Review by tituspijean — APPROVED** [review]

**Pull request merged**